tests/pr_test.go

```diff
@@ -193,6 +195,35 @@ func TestRunCompleteExample(t *testing.T) {
 	}
 }
 
+func TestMultiServiceProfileExample(t *testing.T) {
+	t.Parallel()
+	cloudInfoSvc, err := cloudinfo.NewCloudInfoServiceFromEnv("TF_VAR_ibmcloud_api_key", cloudinfo.CloudInfoServiceOptions{})
+	assert.Nil(t, err, "Failed to create cloud info service")
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  multiServiceExampleTerraformDir,
+		Prefix:        "cbr-multi-service-profile",
+		ResourceGroup: resourceGroup,
+	})
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+
+	outputs := terraform.OutputAll(options.Testing, options.TerraformOptions)
+	zone, err := cloudInfoSvc.GetCBRZoneByID(outputs["zone_id"].([]interface{})[0].([]interface{})[0].(string))
+	rule, err := cloudInfoSvc.GetCBRRuleByID(strings.Split(outputs["rule_id"].([]interface{})[0].(string), ",")[0])
+	assert.Nilf(t, err, "This should not have errored, could not get zone")
+
+	assert.Equal(t, outputs["zone_id"].([]interface{})[0].([]interface{})[0].(string), *zone.ID)
```

Comment:
we also need to check the zone contains the expected serviceRefs (we already had a bug where it didn't)